docs/make.jl

```diff
@@ -66,30 +67,32 @@ end
 
 makedocs(
     modules = [RigidBodyDynamics, RigidBodyDynamics.OdeIntegrators],
-    root = @__DIR__,
+    format = Documenter.HTML(),
     checkdocs = :exports,
-    sitename ="RigidBodyDynamics.jl",
+    sitename = "RigidBodyDynamics.jl",
     authors = "Twan Koolen and contributors.",
     pages = [
         "Home" => "index.md",
         "Tutorials" => tutorialpages,
         "Library" => [
-            "Spatial vector algebra" => "spatial.md",
+            "Spatial Vector Algebra" => "spatial.md",
             "Joints" => "joints.md",
-            "Rigid bodies" => "rigidbody.md",
+            "Rigid Bodies" => "rigidbody.md",
             "Mechanism" => "mechanism.md",
-            "MechanismState" => "mechanismstate.md",
-            "Kinematics/dynamics algorithms" => "algorithms.md",
-            "Custom collection types" => "customcollections.md",
-            "Cache types" => "caches.md",
+            "Mechanism State" => "mechanismstate.md",
+            "Kinematics/Dynamics Algorithms" => "algorithms.md",
+            "Custom Collection Types" => "customcollections.md",
+            "Cache Types" => "caches.md",
             "Simulation" => "simulation.md",
-            "URDF parsing and writing" => "urdf.md",
+            "URDF Parsing and Writing" => "urdf.md",
         ],
         "Benchmarks" => "benchmarks.md",
-      ],
-    format = Documenter.HTML(prettyurls = parse(Bool, get(ENV, "CI", "false")))
+    ],
+    warnonly = Documenter.except(),
```

Comment:
I had to add this line `warnonly = Documenter.except()` because otherwise the build fails due to `:missing_docs` errors (which means not all docstrings are being used by the Docs).